buildbot/master/files/config/factories.py

```diff
@@ -216,6 +235,10 @@ def __init__(self, builder_name, environment):
         ])
 
 
+def is_windows():
+    return platform.system() == "Windows"
```

Comment:
This doesn't work because this code runs on the Buildbot master (Linux). We need to somehow interrogate the builder properties and use that throughout the build. I don't know exactly how to do that.